app/build.gradle

```diff
@@ -26,7 +26,17 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:support-v4:22.0.0'
-    compile 'com.android.support:appcompat-v7:22.0.0'
+    compile 'com.android.support:appcompat-v7:21.0.0'
     compile 'com.mcxiaoke.volley:library:1.0.15'
     compile 'com.google.android.gms:play-services-location:7.0.0'
+
+    compile 'com.android.support:support-v13:21.0.3'
+    compile 'com.android.support:palette-v7:21.0.3'
+    compile 'com.android.support:recyclerview-v7:21.0.3'
+    compile 'com.android.support:cardview-v7:21.0.3'
+    compile 'com.jakewharton:butterknife:6.1.0'
+    compile 'com.jakewharton.timber:timber:2.5.1'
+    compile 'com.facebook.rebound:rebound:0.3.6'
+    compile 'com.squareup.picasso:picasso:2.5.0'
+    compile 'com.commonsware.cwac:camera:0.6.+'
```

Comment:
You don't put a plus sign at the end because it's unstable in case they push a stupid update